samples/corda/corda-simple-application/clients/src/main/resources/config/credentials/network1/access-control.json

```diff
@@ -12,6 +12,30 @@
+      "principalType": "ca",
+      "resource": "localhost:10006#net.corda.samples.tokenizedhouse.flows.GetAssetClaimStatusByPledgeId:*",
+      "read": true
+    },
+    {
+      "principal": "-----BEGIN CERTIFICATE-----\nMIICeDCCAh6gAwIBAgIUQ2X0BIXamzl9FEgyMlwMh/gZm6swCgYIKoZIzj0EAwIw\naDELMAkGA1UEBhMCVVMxFzAVBgNVBAgTDk5vcnRoIENhcm9saW5hMRQwEgYDVQQK\nEwtIeXBlcmxlZGdlcjEPMA0GA1UECxMGRmFicmljMRkwFwYDVQQDExBmYWJyaWMt\nY2Etc2VydmVyMB4XDTIyMDIxNTA0MTIwMFoXDTIzMDIxNTA0MTcwMFowQjEwMA0G\nA1UECxMGY2xpZW50MAsGA1UECxMEb3JnMTASBgNVBAsTC2RlcGFydG1lbnQxMQ4w\nDAYDVQQDEwVhbGljZTBZMBMGByqGSM49AgEGCCqGSM49AwEHA0IABNpzOjnD7koJ\n+hKYLLIL72TvWZhElzMeBD5o567xXUPKzwer7nUBtGehajiYT0AyH7xBC3f+9uFi\nzj0R4PCJ8bSjgcswgcgwDgYDVR0PAQH/BAQDAgeAMAwGA1UdEwEB/wQCMAAwHQYD\nVR0OBBYEFGrx+3f+eZwK2ZUdOQ9/6XnlJ4RDMB8GA1UdIwQYMBaAFHKG9YL6OC86\ny5FIXpqte7bOXSD8MGgGCCoDBAUGBwgBBFx7ImF0dHJzIjp7ImhmLkFmZmlsaWF0\naW9uIjoib3JnMS5kZXBhcnRtZW50MSIsImhmLkVucm9sbG1lbnRJRCI6ImFsaWNl\nIiwiaGYuVHlwZSI6ImNsaWVudCJ9fTAKBggqhkjOPQQDAgNIADBFAiEAhYIdvuR4\nVoaNG4/HBo6pN/LmQ3m0ejJnsQWNqhKTOQQCIAlM09GXobj82tZrbLFAEsnkaJ8C\nmNdFRlJzUAAowGl0\n-----END CERTIFICATE-----\n",
+      "principalType": "ca",
+      "resource": "localhost:10006#com.cordaSimpleApplication.flow.GetAssetPledgeStatusByPledgeId:*",
+      "read": true
+    },
+    {
+      "principal": "-----BEGIN CERTIFICATE-----\nMIICeDCCAh6gAwIBAgIUQ2X0BIXamzl9FEgyMlwMh/gZm6swCgYIKoZIzj0EAwIw\naDELMAkGA1UEBhMCVVMxFzAVBgNVBAgTDk5vcnRoIENhcm9saW5hMRQwEgYDVQQK\nEwtIeXBlcmxlZGdlcjEPMA0GA1UECxMGRmFicmljMRkwFwYDVQQDExBmYWJyaWMt\nY2Etc2VydmVyMB4XDTIyMDIxNTA0MTIwMFoXDTIzMDIxNTA0MTcwMFowQjEwMA0G\nA1UECxMGY2xpZW50MAsGA1UECxMEb3JnMTASBgNVBAsTC2RlcGFydG1lbnQxMQ4w\nDAYDVQQDEwVhbGljZTBZMBMGByqGSM49AgEGCCqGSM49AwEHA0IABNpzOjnD7koJ\n+hKYLLIL72TvWZhElzMeBD5o567xXUPKzwer7nUBtGehajiYT0AyH7xBC3f+9uFi\nzj0R4PCJ8bSjgcswgcgwDgYDVR0PAQH/BAQDAgeAMAwGA1UdEwEB/wQCMAAwHQYD\nVR0OBBYEFGrx+3f+eZwK2ZUdOQ9/6XnlJ4RDMB8GA1UdIwQYMBaAFHKG9YL6OC86\ny5FIXpqte7bOXSD8MGgGCCoDBAUGBwgBBFx7ImF0dHJzIjp7ImhmLkFmZmlsaWF0\naW9uIjoib3JnMS5kZXBhcnRtZW50MSIsImhmLkVucm9sbG1lbnRJRCI6ImFsaWNl\nIiwiaGYuVHlwZSI6ImNsaWVudCJ9fTAKBggqhkjOPQQDAgNIADBFAiEAhYIdvuR4\nVoaNG4/HBo6pN/LmQ3m0ejJnsQWNqhKTOQQCIAlM09GXobj82tZrbLFAEsnkaJ8C\nmNdFRlJzUAAowGl0\n-----END CERTIFICATE-----\n",
+      "principalType": "ca",
+      "resource": "localhost:10006#com.cordaSimpleApplication.flow.GetAssetClaimStatusByPledgeId:*",
+      "read": true
```

Comment:
Can you add policies for Corda_Network2 also, so that we have Corda_Network2 <-> Fabric network1 asset transfer working?
Also may be for Fabric Network2 to both Corda Networks? Just to have all 6 types of interoperability working.
Modify respective files to support that.